stage0/src/apic.rs

```diff
@@ -82,18 +112,11 @@ impl ApicBase {
 ///
 /// See Section 16.12 (x2APIC_ID) in the AMD64 Architecture Programmer's Manual, Volume 2 for more
 /// details.
-pub struct X2ApicIdRegister;
+pub struct X2ApicIdRegister(ApicRegister);
```

Comment:
i personally don't mind this, but afaik in the past we had a convention of not using tuple struct and preferring named properties